documentation/BLOCK_CREATION.md

```diff
@@ -2,7 +2,7 @@
 A step by step guide on creating custom blocks and any "gotchas" you may need to worry about.
```

Comment:
![image](https://github.com/victor-move/exporter-rdc-documentation/assets/101747726/2957895d-cba9-4109-ada6-a9e3e4fb291c)
if they are varying height, are they suppose to be the same height

Reply:
and it looks squished if we put 3 here, kel's figma is at 1200px not 756 in that text

Reply:
Good question, let me give that a try

Reply:
<img width="825" alt="Screen Shot 2023-06-28 at 12 37 23 PM" src="https://github.com/victor-move/exporter-rdc-documentation/assets/99918698/314b9aef-2ae1-4ece-a9e5-108f0b1b7ff8">